test/e2e/multihoming_utils.go

```diff
@@ -130,14 +130,15 @@ func generateNetAttachDef(namespace, nadName, nadSpec string) *nadapi.NetworkAtt
 }
 
 type podConfiguration struct {
-	attachments            []nadapi.NetworkSelectionElement
-	containerCmd           []string
-	name                   string
-	namespace              string
-	nodeSelector           map[string]string
-	isPrivileged           bool
-	labels                 map[string]string
-	requiresExtraNamespace bool
+	attachments                  []nadapi.NetworkSelectionElement
+	containerCmd                 []string
+	name                         string
+	namespace                    string
+	nodeSelector                 map[string]string
+	isPrivileged                 bool
+	labels                       map[string]string
+	requiresExtraNamespace       bool
+	needsIPRequestFromHostSubnet bool
```

Comment:
isn't there a better way to model this without adding yet another boolean ? 

Can't you define here a static ip which you'd pass to the factory func ?

Reply:
Ah I went for a boolean because the logic to pick an IP within the host subnets, which I implemented in `addIPRequestToPodConfig`, can return an error. If I call that when I create the ginkgo.Entry, I'll get a `No Assertions in Container Nodes`.

Reply:
yeah, for that to work, you'd need to pass the function as argument, then execute it in the table's body.

I guess this can be refactored later on ... But the more and more booleans we add here, the harder it'll be. 

Another question: why don't you just hard code an IP ? i.e. just choose something from the range.